docs/source/getting-started/app-architecture.rst

```diff
@@ -0,0 +1,108 @@
+.. Copyright (c) 2020 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+App Architecture
+****************
+
+In this section we'll look at the different components of our social network app.
+The goal is to familiarise you enough to feel comfortable extending the code with a new feature in the next section.
+
+There are two main components in the code - the DAML model and the React/TypeScript frontend - with generated TypeScript code to bridge the two.
+Let's start by looking at the DAML model, as this sets the core logic of the application.
+
+The DAML Model
+==============
+
+Using VSCode (or a code editor of your choice), navigate to the ``daml`` subdirectory.
+There is a single DAML file called ``User.daml`` with the model for users of the app.
+
+The core data is at the start of the ``User`` contract template.
+
+.. literalinclude:: code/daml/User.daml
+  :language: daml
+  :start-after: -- MAIN_TEMPLATE_BEGIN
+  :end-before: -- MAIN_TEMPLATE_END
+
+There are two important aspects here:
+
+1. The data definition (a *schema* in database terms), describing the data stored with each user contract.
+In this case it is an identifier for the user and their current list of friends.
+Both fields use the built-in ``Party`` type which lets us use them in the following clauses.
+
+2. The signatories and observers of the contract.
+The signatories are the parties authorized to create new versions of the contract or archive the contract.
+In this case only the user has those rights.
+The observers are the parties who are able to view the contract on the ledger.
+In this case all friends of a user are able to see the user contract.
+
+Let's say what the ``signatory`` and ``observer`` clauses mean in our app more concretely.
+A user Alice can see another user Bob in the network only when Alice is a friend in Bob's user contract.
+For this to be true, Bob must have previously added Alice as a friend (i.e. updated his user contract), as he is the sole signatory on his user contract.
```

Comment:
I think the word "updated" gives people a wrong sense of how DAML works. I would just remove the parentheses.